src/test/components/ThreadActivityGraph.test.js

```diff
@@ -261,6 +263,29 @@ describe('ThreadActivityGraph', function () {
     expect(getCallNodePath()).toEqual([]);
   });
 
+  it('when clicking a stack, this selects the call tree panel', function () {
```

Comment:
In #4331 I said that I couldn't find existing tests for the activity graph, looks like I found them in this file after all... so I moved the existing test here and added a new one.